.github/workflows/docker-build-release.yaml

```diff
@@ -0,0 +1,39 @@
+name: release-publish
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # GITHUB_REF is tag of the release https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release
+          ref: ${{ github.ref }}
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: devlaunchers/site-projects # list of Docker images to use as base name for tags
+          tag-semver: |
+            {{version}}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push Docker images
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
```

Comment:
Dockerfile is not defined yet.

Reply:
@chungthuang the dockerfile I have in the last [commit](https://github.com/dev-launchers/site-projects/pull/16/commits/d5161093eb91d72aaf92c5cb4bc8f75b1a4efc13) handles environment variables for production. I don't know if it does it correctly but I took it from next.js official docs.